misc/cron/archive.php

```diff
@@ -54,7 +54,7 @@
     $_GET['method'] = 'CoreAdminHome.runCronArchiving';
     $_GET['format'] = 'console'; // will use Content-type text/plain
 
-    if(!isset($_GET['token_auth'])) {
+    if(empty(Piwik\Common::getRequestVar('token_auth', '', 'string'))) {
```

Comment:
```suggestion
    if('' === Piwik\Common::getRequestVar('token_auth', '', 'string')) {
```

As we know that the return value is an empty string if the parameter isn't set in the request, I think it would be more clean to check for that only. `empty` would also check for a couple of more things, which for sure won't hurt, but is simply unneeded.